ansible/ci-playbooks/roles/ci-jenkins-workspace/tasks/linux.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: ci-jenkins-workspace (debian) | Add jenkins logical volume
+  lvol:
+    vg: scratch
+    lv: scratch
+    size: 64512
+  become: yes
+
+- name: ci-jenkins-workspace (debian) | Create scratch filesystem
+  filesystem:
+    # Because it's a scratch filesytem with no important data on it,
+    # disable journalling for performance.
+    fstype: ext4
+    dev: /dev/scratch/scratch
```

Comment:
Definitely needs to be configurable. Can this just use the definition of the `lv` we've already defined above? e.g. could you use ansible-role-lvm-partition, like https://github.com/openmicroscopy/infrastructure/blob/master/ansible/server-state-playbooks/ome-dundeeomero.yml#L130